django_usaid_clinicfinder/clinicfinder/serializers.py

```diff
@@ -22,7 +23,7 @@ class PointOfInterestSerializer(HyperlinkedModelSerializer):
 
     class Meta:
         model = PointOfInterest
-        fields = ('url', 'id', 'data', 'location', 'created_at', 'updated_at')
+        fields = ('url', 'id', 'data', 'location')
```

Comment:
There was issues with the serializer not liking the date object and rather than add the named handler I realised it wasn't in use and removed it